aztec/src/test/kotlin/org/wordpress/aztec/AztecParserTest.kt

```diff
@@ -92,7 +92,7 @@ class AztecParserTest : AndroidTestCase() {
      */
     @Before
     fun init() {
-        context = MockContext()
+        context = ShadowApplication.getInstance().applicationContext
```

Comment:
Hey @loremattei @jtreanor I did spend some time to figure out why MockContext() is crashing during initialization but didn't find a proper reason why that was happening. Instead, I have added `ShadowApplication.getInstance().applicationContext` which is working good.

cc: @hypest